### ISSUE TITLE
[BUG/ISSUE] wrong OH concentration saved in ND51 using v12

### ISSUE THREAD

**Issue by plutologic** [geoschem/geos-chem#600]
I followed the way provided in https://github.com/geoschem/geos-chem/issues/593  to save OH concentration in ND51 using v12.3. But the result seems wrong, OH concentration over China is only 1e-17~1e-14 molec/cm3 in January 2020, much lower than what it should be (1e6 molec/cm3).

**Comment by yantosca** [comment]
Thanks for pointing this out @plutologic.   The units in State_Chm%Species are in v/v and not molec/cm3.  There will have to be a unit conversion.

**Comment by Aydr199406** [comment]
I am a new user of the GEOS-Chem model. Can you give me a contact information such as e-mail address for academic exchange? Thank you very much! @plutologic

**Comment by plutologic** [comment]
> I am a new user of the GEOS-Chem model. Can you give me a contact information such as e-mail address for academic exchange? Thank you very much! @plutologic
yeah, sure, zqq200662@126.com

**Comment by yantosca** [comment]
This issue is now corrected in commits 76814190d370d601e3aeb7fe37754f4dca8d0582 and https://github.com/geoschem/GCClassic/commit/aa3ba6627b626cec2aedc974a2b3071622900218.  The code changes are shown below:

Add this variable:
```F90
       REAL(fp), SAVE    :: CONV_OH
```
Define this after id_OH:
```F90
       id_OH      = Ind_('OH'     )

       ! Used to convert kg/kg dry to molec/cm3
       IF ( id_OH > 0 ) THEN
          CONV_OH = ( AVO / State_Chm%SpcData(id_OH)%Info%Mw_g ) / 1.0e+6_fp
       ENDIF
```

And then the IF statement should be:
```F90
             ELSE IF ( N == 501 .and. IS_FULLCHEM ) THEN

                !--------------------------------------
                ! OH [molec/cm3]
                !--------------------------------------

                ! Accumulate data
                Q(X,Y,K,W) = Q(X,Y,K,W) +                                    &
                     ( State_Chm%Species(I,J,L,id_OH) * GOOD(X) ) *          &
                     ( State_Met%AIRDEN(I,J,L)        * CONV_OH )
```
This needs to be added to `GeosCore/diag51_mod.F90` and `GeosCore/diag51b_mod.F90`.

**Comment by yantosca** [comment]
This will ship in 13.0.0-final. You can add this update to your code in the meantime.  I will close out this issue.
